cli/main.rs

```diff
@@ -52,9 +52,15 @@ fn main() {
 
 fn repl(mut jstime: jstime::JSTime) {
     use rustyline::{error::ReadlineError, Editor};
+    use dirs::home_dir;
+    
+    let mut history_path = home_dir().unwrap();
+    history_path.push(".jstime_repl_history");
 
     let mut rl = Editor::<()>::new();
     println!("Welcome to jstime v{}!", env!("CARGO_PKG_VERSION"));
+    
+    if rl.load_history(&history_path).is_err() {}
```

Comment:
not sure how to better handle this.

`load_history` returns a result and if the file doesn't exist there is an error. This error can actually be ignored as on first run on a system this file will not exist.

Reply:
`let _ = rl.load_history()`